language-server/modules/langserver-core/src/test/resources/completion/new/type_def/config/config1.json

```diff
@@ -1,406 +1 @@
-{
```

Comment:
Config is not formatted, Use indentation of 2 and format

Reply:
Fixed.